lametro/views.py

```diff
@@ -99,6 +99,8 @@ def get_context_data(self, **kwargs):
 
         context['related_bills'] = related_bills
 
+        context['events'] = bill.actions_and_agendas
```

Comment:
Would you mind calling this context variable `actions`, by extension of https://github.com/datamade/la-metro-councilmatic/pull/450#discussion_r498922714?